seminar_1/app/src/main/java/org/sopt/androidseminar/SignUpActivity.kt

```diff
@@ -0,0 +1,80 @@
+package org.sopt.androidseminar
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.Toast
+import org.sopt.androidseminar.databinding.ActivitySignupBinding
+
+class SignUpActivity : AppCompatActivity() {
+    private val TAG:String= "SignUpActivity";
+    private lateinit var binding: ActivitySignupBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Log.d(TAG, "onCreate")
+        binding = ActivitySignupBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        initButtonClickEvent()
+
+    }
+
+    private fun initButtonClickEvent() {
+        binding.btnSignUp.setOnClickListener {
+
+            val userName: String = binding.etRegisterName.text.toString()
+            val userId: String = binding.etRegisterId.text.toString()
+            val userPw: String = binding.etRegisterPw.text.toString()
+
+            if (userName.isNullOrBlank() || userId.isNullOrBlank() || userPw.isNullOrBlank()) {
+                Toast.makeText(
+                    this@SignUpActivity,
+                    "빈 칸이 있는지 확인해주세요", Toast.LENGTH_LONG
+                ).show()
+            } else {
+                val result = Intent()
+                result.putExtra("regiName", userName)
+                result.putExtra("regiId", userId)
+                result.putExtra("regiPw", userPw)
```

Comment:
사실 이거는 정말정말정말 미세먼지 팁이지만 하나 언급해 드리자면, 이런 식으로 적으셔도 되지만,
```suggestion
                result.putExtra("regiName", userName)
                .putExtra("regiId", userId)
                .putExtra("regiPw", userPw)
```

이렇게도 쓸 수 있어요 :)